client/my-sites/site-settings/section-import.jsx

```diff
@@ -30,10 +29,41 @@ import Main from 'components/main';
 import HeaderCake from 'components/header-cake';
 import Placeholder from 'my-sites/site-settings/placeholder';
 
-const hasActiveImports = imports =>
-	!! find( imports, ( { importerState } ) => {
-		return importerState !== appStates.INACTIVE && importerState !== appStates.READY_FOR_UPLOAD;
-	} );
+/**
+ * Configuration for each of the importers to be rendered in this section. If
+ * you're adding a new importer, add it here. Importers will be rendered in the
+ * order they are listed in this array.
+ *
+ * @type {Array}
+ */
+const importers = [
+	{
+		type: WORDPRESS,
+		isImporterEnabled: true,
+		component: WordPressImporter,
```

Comment:
> Variable and function names should be full words, using camel case with a lowercase first letter. This also applies to abbreviations and acronyms.
> [...]
> Constructors intended for use with new should have a capital first letter (UpperCamelCase).

Since this is a prop of an object, the rules for variables apply. 

( from https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md )